docs/docs/10.1-animation.md

```diff
@@ -86,7 +87,7 @@ You'll notice that animation durations need to be specified in both the CSS and
 
 `ReactCSSTransitionGroup` provides the optional prop `transitionAppear`, to add an extra transition phase at the initial mount of the component. There is generally no transition phase at the initial mount as the default value of `transitionAppear` is `false`. The following is an example which passes the prop `transitionAppear` with the value `true`.
 
-```javascript{3-5}
```

Comment:
I made it up :) http://zpao.com/posts/adding-line-highlights-to-markdown-code-fences/

As for why… it's usually to highlight a specific change or something important - that's the main way we use it in the tutorial. Most of the uses in here… not actually that valuable. The first on the page one is probably the most valuable since it's a larger code block.